File: Leetcode/L_Sorting_the_Sentence.py
# Sorting the Sentence
# String Sorting

class Solution:
    def sortSentence(self, s: str) -> str:
        str1 = ""
        X = list(s.split(" "))
        for i in range(0, len(X)):
            X[i] = X[i][::-1]
        X.sort()
        for i in range(0, len(X)):
            X[i] = X[i][::-1]
            X[i] = ''.join([i for i in X[i] if not i.isdigit()])
        for i in range(0, len(X) - 1):
            X[i] += " "
        for i in range(0, len(X)):
            str1 += X[i]
        return str1


X = Solution()
print(X.sortSentence("is2 sentence4 This1 a3"))
print(X.sortSentence("Myself2 Me1 I4 and3"))

File: Leetcode Js/L_Sorting_the_Sentence.js
// Sorting the Sentence
// String, Sorting, Js

/**
 * @param {string} s
 * @return {string}
 */
var sortSentence = function (s) {
  var [str1, i, j] = ["", 0, 0];
  var X = Array.from(s.split(" "));
  for (i = 0; i < X.length; i++) {
    X[i] = X[i].split("").reverse().join("");
  }
  X.sort();
  for (i = 0; i < X.length; i++) {
    X[i] = X[i].split("").reverse().join("");
    X[i] = X[i].replace(/[0-9]/g, "");
  }
  X = X.join(" ");
  return X;
};

console.log(sortSentence("is2 sentence4 This1 a3"));
console.log(sortSentence("Myself2 Me1 I4 and3"));
